src/schema/v2/schema.ts

```diff
@@ -238,6 +238,7 @@ import { DiscoverArtworks } from "./discoverArtworks"
 import { CreateDiscoveryLikedArtworkMutation } from "./createDiscoveryArtworkReferenceMutation"
 import { Invoice } from "./Invoice/invoice"
 import { createInvoicePaymentMutation } from "./Invoice/createInvoicePaymentMutation"
+import { ackTaskMutation } from "./me/ack_task_mutation"
```

Comment:
Nitpick: can we change the name of the file to follow a Camel Case convention? just to be similar to the other files 👀? 
```suggestion
import { ackTaskMutation } from "./me/ackTaskMutation"
```

Reply:
there's plenty of others that use this convention, including the dismiss mutation. Is the recommended format CamelCase? Should I change the dismiss mutation as well?

Reply:
Maybe that would be great to follow one convention 😅 but it's not part of this PR's scope, so feel free to ignore the other ones